src/sagemaker/amazon/factorization_machines.py

```diff
@@ -21,7 +21,8 @@
 
 class FactorizationMachines(AmazonAlgorithmEstimatorBase):
 
-    repo = 'factorization-machines:1'
+    alg_name = 'factorization-machines'
```

Comment:
I would still name these "repo_name" and "repo_tag" or similar, since that's what they are first and foremost.